claimManagement/build.gradle

```diff
@@ -16,7 +16,51 @@ android {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
+        debug {
+            debuggable = true
+        }
+        flavorDimensions 'std'
+    }
+    flavorDimensions 'std'
+
+    flavorDimensions 'std'
+    productFlavors.all {
+        buildConfigField "String", "API_BASE_URL", '"http://demo.openimis.org/rest/"'
+        buildConfigField "String", "RAR_PASSWORD", '")(#$1HsD"'
     }
+    productFlavors {
+        demoProd {
+            buildConfigField "String", "API_BASE_URL", '"http://demo.openimis.org/rest/"'
+            dimension = 'std'
+        }
+
+        demoRelease {
+            buildConfigField "String", "API_BASE_URL", '"http://release.openimis.org/rest/"'
+        }
+
+        chfDev {
+            buildConfigField "String", "API_BASE_URL", '"http://chf-dev.swisstph-mis.ch/rest/"'
+            dimension = 'std'
+        }
+        mvDev {
+            buildConfigField "String", "API_BASE_URL", '"http://imis-mv.swisstph-mis.ch/rest/"'
+            dimension = 'std'
+        }
+        bephaDev {
+            buildConfigField "String", "API_BASE_URL", '"http://149.210.235.40/devupgrade/rest/"'
+            dimension = 'std'
+        }
+        local {
+            buildConfigField "String", "API_BASE_URL", '"http://10.0.2.2:35788/"'
+            dimension = 'std'
+        }
+    }
+
+    sourceSets {
+        chfDev.res.srcDir 'src/chf/res'
```

Comment:
Do we have escape procedures for ValidateCHF? Do we need to have specific java.srcDir here?

Reply:
I added source sets with Escape.java for every build variant.